### PR TITLE
Make Website Accessible 

### PR DESCRIPTION
**Pull request by waseemqureshi2020**
Here are the changes I have made.

Page title --- No Issue Found.
Image text alternative --- 4 Issues Found And Fixed.
Text heading --- No Issue Found.
Color contrast --- No Issue Found.
Resize --- 1 Issue Found And Fixed.
Interactions --- 3 Issues Found (Optional).
Multimedia --- No Issue Found.
Basic Structure --- No Issue Found.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -189,12 +188,13 @@ <h2 class="about-me-heading">About me</h2>
...
-        <div class="about-me-image">
-            <img id="abc" src="images/Frame.png" alt="about me">
-        </div>
     </div>
     <div class="divider"></div>
+    </section>
+
+   <!-- Desktop Languages Section -->
 
+    <section class="languages-section">
```
Comment (christianbiring1) [inline index.html:197]: [ OPTIONAL]
- [ ] Kindly consider using the same color for the elements as the ones provided in Figma.
- [ ] Kindly make sure the correct spacing `margin/padding` is used to place your element on the page as it is the case Figma template.
Your design :point_down: 
image

Figma design :point_down: 
image

File: index.html [hunk 2]
```diff
@@ -257,7 +257,7 @@ <h2 id="l1" class="about-me-languages">Language</h2>
             </ul>
         </div>
         <div class="desktop-skills-content">
-            <img id="rect2" src="images/Rectangle 56.png">
+            <img id="rect2" src="images/Rectangle 56.png" alt="shape 2">
```
Comment (christianbiring1) [inline index.html:260]: - [ ] Kindly consider using a descriptive `alternative text` for your images, That tells what the image is about.

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @waseemqureshi2020 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose

## Highlights
- [ ] Descriptive PR `title` and `summary`
- [ ] All linters checks are green :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @waseemqureshi2020 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:

- [ ] As the previous reviewer asked, kindly try to follow the Figma design as much as possible, this includes text alignment and spacing for these sections:

## The header:
### Your design:
image

### Figma design:
image

## The projects cards (they should be centered)
### Your design:
image

### Figma design:
image

## Contact section: the text should in the center of the page
### Your design:
image

### Figma design:
image



------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -29,7 +29,7 @@
     <div class="name-desktop "><h1 class="desktop-h1 animate-logo">Waseem Qureshi</h1></div>
     <nav class="desktop-nav">
         <ul class="desktop-list">
-            <li class="desktop-item"><a class="top-links" href="#"><img src="images/Union.png"></a></li>
+            <li class="desktop-item"><a class="top-links" href="#"><img src="images/Union.png" alt="illustration"></a></li>
```
[the remark this answers is not in the document]
Comment (waseemqureshi2020) [reply index.html:32]: I have changed the position of the header elements. I have also changed the font size, font family, weight, and spacing.

File: index.html (hunk 1 above)
Comment (waseemqureshi2020) [reply index.html:197]: I have changed the color and aligned the item in the center.

File: index.html (hunk 2 above)
Comment (waseemqureshi2020) [reply index.html]: Thank you for pointing this out. I have changed the alt text for these shapes.

**Pull request merged**